include/libnuraft/raft_params.hxx

```diff
@@ -606,7 +607,22 @@ public:
      */
     bool parallel_log_appending_;
 
+    /**
+     * It great than 0, append entry request will be sent immediately without
+     * waiting for the response of previous append entry request.
+     * 
+     * The throtlling is control by max_log_gap_in_stream_ and max_flying_bytes_.
+     * 
+     * If the number of flying log entry exceeds max_log_gap_in_stream_ 
+     * or flying bytes exceeds max_flying_bytes_, 
+     * next request will not be delivered immediately.
+    */
     int32 max_log_gap_in_stream_;
+
+    /**
+     * Max flying bytes we allow. If it is zero, we don't use it as throttling.
+    */
+    ulong max_flying_bytes_;
```

Comment:
This parameter will be used for streaming mode only right? then `max_bytes_in_flight_in_stream_`.